coord/src/module.c

```diff
@@ -1966,7 +1986,20 @@ static int initSearchCluster(RedisModuleCtx *ctx, RedisModuleString **argv, int
                   clusterConfig.type, clusterConfig.timeoutMS);
 
   if (clusterConfig.type == ClusterType_RedisOSS) {
+    // Check if we are actually in cluster mode
+    RedisModuleCallReply *rep = RedisModule_Call(ctx, "CONFIG", "cc", "GET", "cluster-enabled");
```

Comment:
Did you check that this configuration is populated before the module load?

Reply:
Yes. We also see that both cases of the following condition on the configuration values are covered by tests